packages/front-end/pages/features/index.tsx

```diff
@@ -336,6 +350,19 @@ export default function FeaturesPage() {
                         />
                       </td>
                     )}
+                    <td>
+                      <MoreMenu id={`more-menu-feature-${feature.id}`}>
```

Comment:
`feature.id` can have non-spec characters here

Reply:
Latest change: Removes id prop on MoreMenu since it is now internally maintained by the component (uniquely generated id via uniqid)